cocos/ui/UIRichText.cpp

```diff
@@ -373,12 +801,23 @@ void RichText::handleTextRenderer(const std::string& text, const std::string& fo
     }
 }
     
-void RichText::handleImageRenderer(const std::string& fileParh, const Color3B &color, GLubyte opacity)
+void RichText::handleImageRenderer(const std::string& filePath, const Color3B &color, GLubyte opacity, int width, int height)
 {
-    Sprite* imageRenderer = Sprite::create(fileParh);
+    Sprite* imageRenderer = Sprite::create(filePath);
```

Comment:
I suggest to use Scale9Sprite with `RenderingType::Simple` here. The code will be more compact. We don't need to calculate the scale.